user_templates/mcu_defaults/main/Mellow_Fly_Gemini_v3.cfg

```
@@ -1,15 +1,11 @@
 
 #------------------------------------------#
-#### CUSTOM TEMPLATE MCU definition ########
+#### Mellow Fly Gemini V3 MCU definition ###
 #------------------------------------------#
 
-# This template file is a pre-filled file with Klippain pins alias conventions
-# that can be used if your MCU board is not yet officially supported. Just fill
-# in your MCU pins and you will be good to go :)
-
 [mcu]
 ##--------------------------------------------------------------------
-serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+serial: /dev/serial/by-id/usb-Klipper_stm32f405xx_JKYZ-if00
```

Please keep here the dummy placeholder:

```suggestion
serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
```

---

no problem, will you resolve the other ones?